applications/ColossalEval/colossal_eval/dataset/safetybench_zh.py

```diff
@@ -0,0 +1,137 @@
+import copy
+import csv
+import os
+from typing import Dict, List
+import json
```

Comment:
Please refer to `applications/ColossalEval/colossal_eval/dataset/safetybench_en.py` for comments.